dataactbroker/handlers/accountHandler.py

```diff
@@ -558,23 +560,33 @@ def resetPassword(self,system_email,session):
         except Exception as e:
             exc = ResponseException("Unknown Error",StatusCode.CLIENT_ERROR,ValueError)
             return JsonResponse.error(exc,exc.status)
+
+        email = requestDict.getValue("email")
+        LoginSession.logout(session)
+        self.sendResetPasswordEmail(user, system_email, email)
+
+        # Return success message
+        return JsonResponse.create(StatusCode.OK,{"message":"Password reset"})
+
+    def sendResetPasswordEmail(self, user, system_email, email=None, inactive_user_override=False):
```

Comment:
Minor nitpick, I feel like parts of this function would be clearer if the last flag were named something like unlock_user rather than inactive_user_override, especially where we use it to pick the email template for unlocking a user